Apps/Database/src/Models/UserProfile.cs

```diff
@@ -72,5 +73,10 @@ public class UserProfile : AuditableEntity
         /// </summary>
         [MaxLength(44)]
         public string? EncryptionKey { get; set; }
+
```

Comment:
We don’t need this at this point.

Reply:
Do you mean the empty line or the EncryptionKey  ?

Reply:
removing virtual ... CommunicationEmails